src/UI/Component/Modal/Interruptive.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+/**
+ * Interface Interruptive
+ *
+ * @package ILIAS\UI\Component\Modal
+ */
+interface Interruptive extends Modal {
+
+	/**
+	 * Get the message of this modal, displayed below the modals title
+	 *
+	 * @return string
+	 */
+	public function getMessage();
+
+
+	/**
+	 * Get the title of this modal
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+
+	/**
+	 * Get a modal like this with the given title
+	 *
+	 * @param string $title
+	 *
+	 * @return Interruptive
+	 */
+	public function withTitle($title);
```

Comment:
Do we have the use case where we change the title of the interruptive modal after creation?